scripts/dictionary.txt

```diff
@@ -90,3 +90,6 @@ Neovim
 nvim-treesitter
 Treesitter's
 parsers
+renderable
+(R|r)ehydration
```

Comment:
I feel like we should avoid rehydration and hydration as they mean something specific to React and could have their definition changed out fro under us.